web/src/main/java/org/cbioportal/web/parameter/StudyViewFilter.java

```diff
@@ -12,10 +13,11 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import org.cbioportal.model.AlterationFilter;
 import org.cbioportal.model.GeneFilter;
+import org.cbioportal.model.StudyViewStructuralVariantFilter;
 import org.cbioportal.web.parameter.filter.*;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
-@JsonInclude(Include.NON_NULL)
+@JsonInclude(Include.ALWAYS)
```

Comment:
Why do we need to change this? To prevent null pointer exceptions?

Reply:
Reverted in [9665e46](https://github.com/cBioPortal/cbioportal/pull/10214/commits/9665e463a464f5506e36ab3f1d7130b377f8f059)